src/dagre-js/shapes.js

```diff
@@ -4,12 +4,21 @@ import { intersectPolygon } from "./intersect/intersect-polygon";
 import { intersectRect } from "./intersect/intersect-rect";
 
 export {
+  shapes,
+  setShapes
+};
+
+var shapes = {
```

Comment:
Do you mind using `const` instead of `var`?

Reply:
https://github.com/tbo47/dagre-es/pull/3#discussion_r1014727984